google/cloud/forseti/services/inventory/storage.py

```diff
@@ -348,18 +348,18 @@ def get_schema_update_actions():
         Returns:
             dict: A mapping of Action: Column.
         """
-        columns_to_create = [Column('cai_resource_type',
-                                    String(512),
-                                    default=''),
-                             Column('cai_resource_name',
-                                    String(4096),
-                                    default=''),
-                             Column('full_name',
-                                    String(2048),
-                                    nullable=False)]
-
-        schema_update_actions = {'CREATE': columns_to_create}
-        return schema_update_actions
+        columns_to_alter = {
```

Comment:
Adding the gcp_inventory.category column to the DB migration because new categories have been added for Org/Access policy.